PubnativeLite/PubnativeLite/MRAID/HyBidMRAIDView.m

```diff
@@ -552,7 +556,14 @@ - (void)expand:(NSString *)urlString {
     
     if (!urlString) {
         // 1-part expansion
-        webView.frame = frame;
+        CGFloat height = 480;
```

Comment:
Wouldn't it be better to put these variables inside the **if**

Reply:
actually that didn't affect!
Anyway, those static variables will be dynamic when we will support iPads or even better if we could get it from Backend.

Reply:
Wouldn't it better if we change the frame of the `modalVC`? So that we can use the size that we pass when we create the `HyBidMRAIDView`? + it'll be dynamic as well.

Reply:
Would be cool but would we still have the frame though? Won't we just see the underlying VC?

Reply:
but then we will have a transparent background if we change the frame of the modal VC. The idea is to stay fullscreen but manage the content.

Reply:
@eros902002 I've meant modalVC's view. Sorry. like (`modalVC.view.frame`)

Reply:
@faresbh there's a `center` property that you can use also. It is a `CGPoint` 😉

Reply:
I tried that! it uses the frame UIView but in my case, I want the Screen size, to avoid safe layouts mismatching with some devices.